check-windows-eventlog/check-windows-eventlog.go

```diff
@@ -293,6 +371,30 @@ func (opts *logOpts) searchLog(logName string) (warnNum, critNum int64, errLines
 		}
 		lastNumber = r.RecordNumber
 
+		if opts.idRangeList != nil {
+			// even code takes last 4 bytes
+			eventID := r.EventID & 0x0000FFFF
+			found := false
+			exact := false
+			for _, idr := range opts.idRangeList {
+				if !idr.bang {
+					exact = true
+					if idr.lo <= eventID && eventID <= idr.hi {
+						found = true
+					}
+				} else {
+					if idr.lo <= eventID && eventID <= idr.hi {
+						found = false
+					} else if !exact {
+						found = true
```

Comment:
`!2,!3` against `2` maybe become `found = true`, though `!3,!2` against `2` maybe become `found = false`. Is this behavior accepted? (imo acceptable...)